docs/installation.md

```diff
@@ -0,0 +1,47 @@
+## Wordpress Installation using Boiler Plate
+1. Clone this repository.
+```sh
+git clone https://github.com/ColoredCow/wordpress-init.git
+```
+2. On your local machine, install composer in the root directory using the CLI.
+```sh
+composer install
+```
+3. Run `npm install` in ***/public/wp-content/themes/ColoredCow/*** directory.
+4. Run `grunt` to check if the grunt installation worked. You should see a style.css and main.js inside your theme.
+5. Secure your WordPress installation.
+   * Make a copy of `wp-sample-config.php` in *wordpress-init/public/* directory and name it as `wp-config.php`.
```

Comment:
can you also update the `wp-config.php` and remove the `WP_ENVIRONMENT` variable from there?

Reply:
Sure, made changes in the latest commit.